.circleci/config.yml

```diff
@@ -7,71 +7,18 @@ orbs:
 jobs:
   validate:
     docker:
-      - image: quay.io/giantswarm/opsctl
-        auth:
-          username: $DOCKER_USER
-          password: $DOCKER_PASS
+      - image: quay.io/giantswarm/architect
     steps:
       - checkout
       - run:
-          name: Validate KVM releases
-          command: opsctl validate releases -p kvm.yaml
```

Comment:
@giantswarm/sig-releng is this only place we doing for `opsctl validate`?

Reply:
Yes only here and occasionally manually.